content/general/concepts/geolocation/geolocation.md

```diff
@@ -0,0 +1,25 @@
+---
+Title: 'Geolocation'
+Description: 'Geolocation refers to the process of determining the geographic location of an object, typically through electronic means such as GPS.'
+Subjects:
+  - 'Computer Science'
+  - 'Mobile Development'
+Tags:
+  - 'Map'
+CatalogContent:
+  - 'paths/code-foundations'
+  - 'paths/computer-science'
+---
+
+**Geolocation** refers to the process of determining the geographic location of an object, typically through electronic means such as GPS. The most common example of geolocation currently is the location function in various electronic devices such as smartphones.
+
+Many modern applications can be given access to a device's geolocation data, so a review app can show restaurants within a few miles, a banking app can show nearby branches, or a game can offer [augmented reality](https://www.codecademy.com/resources/docs/general/augmented-reality) experiences based on where the device is located.
+Also, most smartphones have a mapping app that can locate the phone on a street map in order to provide directions to some other location.
+
+## Common Types of Geolocation
```

Comment:
I don't know if this header accurately summarizes what's going on in the section. This is describing three _"ways"_ of accessing geolocation services; not necessarily "types". What about something like "Using Geolocation Services"?